test/git/duo/author_collection_test.rb

```diff
@@ -21,6 +21,16 @@ def test_where_partial_match
       assert_equal "Alfred Pennyworth", result.name
     end
 
+    def test_closest_partial_match
```

Comment:
I've been using the pattern `def test_METHOD_EDGECASE`. Could you rename this to `def test_where_*` to follow that?